ctapipe_io_magic/tests/test_lidar.py

```diff
@@ -0,0 +1,48 @@
+import os
+import numpy as np
+from pathlib import Path
+
+import pytest
+
+test_data = Path(os.getenv("MAGIC_TEST_DATA", "test_data")).absolute()
+test_calibrated_real_dir = test_data / "real/calibrated"
+test_calibrated_real = [
+    test_calibrated_real_dir / "20210314_M1_05095172.002_Y_CrabNebula-W0.40+035.root",
+]
+
+data_dict = dict()
+
+data_dict["20210314_M1_05095172.002_Y_CrabNebula-W0.40+035.root"] = dict()
+
+data_dict["20210314_M1_05095172.002_Y_CrabNebula-W0.40+035.root"]["n_events_tot"] = 500
+data_dict["20210314_M1_05095172.002_Y_CrabNebula-W0.40+035.root"]["n_events_stereo"] = 452
+data_dict["20210314_M1_05095172.002_Y_CrabNebula-W0.40+035.root"]["n_events_pedestal"] = 48
+data_dict["20210314_M1_05095172.002_Y_CrabNebula-W0.40+035.root"]["n_events_mc_mono"] = 0
+
+def test_sample_report_laser(sample_report_laser):
+    assert sample_report_laser is not None
+
+@pytest.fixture
+def sample_report_laser():
+    from ctapipe_io_magic import MAGICEventSource, ReportLaserContainer
+    dataset = (test_calibrated_real_dir/"20210314_M1_05095172.002_Y_CrabNebula-W0.40+035.root")
+    with MAGICEventSource(input_url=dataset) as source:
+        print(ReportLaserContainer())
+        return ReportLaserContainer()
+
+@pytest.mark.parametrize("dataset", test_calibrated_real)
+def test_lidar_parameters(dataset):
+    from ctapipe_io_magic import MAGICEventSource
+    with MAGICEventSource(input_url=dataset) as source:
+        laser = source.laser
+        for key, report_list in laser.items():
```

Comment:
currently laser is defined as a dict (right?), so when you make items you will get a list of key, value, therefore this is very misleading